### DISCUSSION TITLE
copy and paste scripts

### DISCUSSION THREAD

**Discussion by FloFree** [dbeaver/dbeaver#19828] [DBeaver Development]
When I try to copy and paste a script file I lose connection to the associated datasource, no datasource is reassociated in the duplicate file.
In the image shown you can see how the "Script.sql" file is associated with the "postgres" datasource, if I try to copy and paste the duplicate file it has no datasource.

![script](https://user-images.githubusercontent.com/14328336/234984921-b066c0dd-dc75-4d21-a71d-de5f866a0662.png)

In the old version 5 the files kept the association, but from 6 onwards this doesn't happen anymore.
Is there any setting that needs to be turned on?

**Comment by LonwoLonwo** [comment]
Hello @FloFree 

Do you copy and paste scripts inside one project or in different projects?

**Reply by FloFree** [reply]
I don't understand what you mean (you can see the image, I think the project is the default one)
